HotKey/HotKey/Form1.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Diagnostics;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace HotKey
+{
+    public partial class Form1 : Form
+    {
+        KeyboardHook hook = new KeyboardHook();
+
+        public Form1()
+        {
+            InitializeComponent();
+
+            // register the event that is fired after the key press.
+            hook.KeyPressed += new EventHandler<KeyPressedEventArgs>(hook_KeyPressed);
+            // register the control +  F12
+            hook.RegisterHotKey((ModifierKeys)2, Keys.F12);
+            hook.RegisterHotKey((ModifierKeys)2, Keys.F11);
+
+        }
+
+        void hook_KeyPressed(object sender, KeyPressedEventArgs e)
```

Comment:
shouldn't there be a hook for every key bind, rather that one for all, so that the logic for deciphering which key isn't done in the hook?

Reply:
I looked into it for now, let's keep it as is for now. We are going to need to think of a more dynamic way of storing them